lib/components/app/responsive-webapp.js

```diff
@@ -268,24 +256,18 @@ class RouterWrapperWithAuth0 extends Component {
               // For new users, it displays the account setup form.
               // For existing users, it takes the browser back to the itinerary search prior to login.
               path={'/signedin'}
-              component={routerProps => {
-                const props = this._combineProps(routerProps)
-                return <AfterSignInScreen {...props} />
-              }}
+              component={AfterSignInScreen}
             />
             <Route
               path='/print'
-              component={routerProps => {
-                const props = this._combineProps(routerProps)
-                return <PrintLayout {...props} />
-              }}
+              component={PrintLayout}
```

Comment:
This causes the error message below and more:
```log
Warning: Failed prop type: The prop `LegIcon` is marked as required in `PrintableItinerary`|`AccessLeg`, but its value is `undefined`.
    in PrintableItinerary (created by PrintLayout)
    in PrintLayout (created by ConnectFunction)
    in ConnectFunction (created by Route)
    ...
```

Reply:
PrintLayout should be receiving the LegIcon via context. Changes inbound to fix this.

Reply:
Fixed in bbb6987f3a529c8851d6d7e868fabade4632872d.